yb-voyager/src/tgtdb/oracle.go

```diff
@@ -243,37 +264,105 @@ outer:
 	return nil
 }
 
-func (tdb *TargetOracleDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, numChans int, startClean bool) error {
+func (tdb *TargetOracleDB) clearLiveMigrationMetaInfo(conn *sql.Conn, metainfoTable string, migrationUUID uuid.UUID) error {
+	startCleanStmt := fmt.Sprintf("DELETE FROM %s where migration_uuid='%s'", metainfoTable, migrationUUID)
+	res, err := conn.ExecContext(context.Background(), startCleanStmt)
+	if err != nil {
+		return fmt.Errorf("error executing stmt - %v: %w", startCleanStmt, err)
+	}
+	rowsAffected, _ := res.RowsAffected()
+	log.Infof("deleted existing channels meta info using query %s; rows affected = %d", startCleanStmt, rowsAffected)
+	return nil
+}
+
+func (tdb *TargetOracleDB) getLiveMigrationMetaInfo(conn *sql.Conn, migrationUUID uuid.UUID) (int64, error) {
+	rowsStmt := fmt.Sprintf(
+		"SELECT count(*) FROM %s where migration_uuid='%s'", EVENT_CHANNELS_METADATA_TABLE_NAME, migrationUUID)
+	var rowCount int64
+	err := conn.QueryRowContext(context.Background(), rowsStmt).Scan(&rowCount)
+	if err != nil {
+		return 0, fmt.Errorf("error executing stmt - %v: %w", rowsStmt, err)
+	}
+	return rowCount, nil
+}
+
+func (tdb *TargetOracleDB) getLiveMigrationMetaInfoByTable(conn *sql.Conn, migrationUUID uuid.UUID, tableName string) (int64, error) {
```

Comment:
Looks like all the changes in this file are copy of equivalent code in yugabytedb.go. So the same comments apply here as well.